genanki/model.py

```diff
@@ -8,13 +8,14 @@ class Model:
   FRONT_BACK = 0
   CLOZE = 1
 
-  def __init__(self, model_id=None, name=None, fields=None, templates=None, css='', model_type=FRONT_BACK):
+  def __init__(self, model_id=None, name=None, fields=None, templates=None, css='', model_type=FRONT_BACK, sort_field_number=1):
     self.model_id = model_id
     self.name = name
     self.set_fields(fields)
     self.set_templates(templates)
     self.css = css
     self.model_type = model_type
+    self.sort_field_id = sort_field_number-1
```

Comment:
So this line becomes `self.sort_field_index = sort_field_index`.

Note that `self.sort_field_id = sort_field_number-1` is incompatible with a general API principle of genanki, which is that
```
c = Class(foo='bar')
```
usually does the same thing as
```
c = Class()
c.foo = 'bar'
```